src/components/PokemonsList/PokemonsList.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import Search from "../Search/Search";
+import Pokemon from "../Pokemon/Pokemon";
+
+const Section = styled.section`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    .wrapper {
+        display: flex;
+        flex-wrap: wrap;
+        justify-content: center;
+    }
+    `
+
+class PokemonsList extends Component {
+    state = {
+        pokemons: [],
+        searchTerm: ""
+    };
+
+    componentDidMount() {
+        fetch('http://pokeapi.co/api/v2/pokemon/')
+            .then(response => response.json())
+            .then(data => this.setState({ pokemons: data.results })
+        );
+    }
+
+    handleSearchInput = event => {
```

Comment:
`handleSearchInput = ({target: {value}}) =>`
i niżej 
` this.setState({ searchTerm: value });`